docs/api.asciidoc

```diff
@@ -299,7 +298,7 @@ TraceContext returns the span's <<trace-context, trace context>>.
 [[apm-context-with-span]]
 ==== `func ContextWithSpan(context.Context, *Span) context.Context`
 
-ContextWithSpan adds the span to the context, and returns the resulting context.
```

Comment:
Span is inconsistently capitalized throughout this whole doc. It's capitalized and it's not. Which is it and then I can update?

Reply:
So in this use case, it would be capitalized?